lib/rubocop/rspec/cop/generator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module RSpec
+    module Cop
+      # Source and spec generator for new cops
+      #
+      # This generator will take a cop name and generate a source file
+      # and spec file when given a valid qualified cop name.
+      # @api private
+      class Generator < RuboCop::Cop::Generator
+        def todo
+          <<~TODO
+            Do 4 steps:
+              1. Modify the description of #{badge} in config/default.yml
+              2. Implement your new cop in the generated file!
+              3. Commit your new cop with a message such as
+                 e.g. "Add new `#{badge}` cop"
```

Comment:
4 and 3 feel out of order. For our parent project it’s justifiable, they need the pr number in the changelog entry? For us, changelog can be added first if you follow the changelog-driven development approach 😅